scripts/zones/Mhaura/npcs/Felisa.lua

```diff
@@ -11,7 +11,13 @@ end
 entity.onTrigger = function(player, npc)
 
     if (player:getZPos() > 38.5) then
-        player:startEvent(221, player:getGil(), 100)
+        if player:getQuestStatus(OTHER_AREAS_LOG, tpz.quest.id.otherAreas.HIS_NAME_IS_VALGEIR) == QUEST_ACCEPTED and
```

Comment:
indentation, and changing log reference:

```
if
    player:getQuestStatus(tpz.quest.log_id.OTHER_AREAS, tpz.quest.id.otherAreas.HIS_NAME_IS_VALGEIR) == QUEST_ACCEPTED and
    player:hasKeyItem(tpz.ki.ARAGONEU_PIZZA)
then
```

Reply:
Definitely on the formatting.  Can you clarify on the log reference?  `OTHER_AREAS_LOG` is specific to the table, whereas this is passing an int value.  `tpz.quest.log_id` only has an `OTHER_AREAS` reference.

Reply:
In the code you're adding in this PR, `if player:getQuestStatus(OTHER_AREAS_LOG, ...` is passing OTHER_AREAS_LOG, which is table.

In my suggested code, `player:getQuestStatus(tpz.quest.log_id.OTHER_AREAS, ...` is passing an int (4), which the binding should accept.

Reply:
Doh! I thought I fixed that.  Absolutely correct.